### PR TITLE
Add first draft of email overview

### PR DESCRIPTION
**Pull request by pauldoomgov**
Provides a public friendly overview of email flow in and out of
login.gov.

### CODE REVIEW

**Comment by zachmargolis** [comment]
This was branched off `master` but the repo has moved to `main`... I'm going to rebase this and force push for ya

File: _articles/infrastructure-email.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+- [Outbound Sending from @login.gov](#outbound-sending-from-logingov)
+  - [GMail Out](#gmail-out)
+  - [AWS SES](#aws-ses)
+  - [Sender Policy Framework (SPF)](#sender-policy-framework-spf)
+  - [DomainKeys Identified Email (DKIM)](#domainkeys-identified-email-dkim)
+  - [Domain-based Message Authentication, Reporting, and Conformance (DMARC)](#domain-based-message-authentication-reporting-and-conformance-dmarc)
+- [Querying Authoritative DNS Servers](#querying-authoritative-dns-servers)
+- [Further Reading](#further-reading)
+
+<!-- /TOC -->
```
Comment (zachmargolis) [inline _articles/infrastructure-email.md]: The handbook auto-generates a table of contents from headings, you can remove this

I did notice that most of the headings are H1, so I'm going to push a commit that moves them to h2 and beyond

**Review by zachmargolis — APPROVED** [review]
LGTM

File: _articles/infrastructure-email.md (hunk 1 above)
Comment (zachmargolis) [reply _articles/infrastructure-email.md]: see 58fe6b82f391c3b595c3a6a43d38c71ee32d211b

**Comment by pauldoomgov** [comment]
Ack... Thanks @zachmargolis !

**Pull request merged**